experiment/run_experiment.py

```diff
@@ -481,27 +482,45 @@ def main():
                         '--experiment-name',
                         help='Experiment name.',
                         required=True)
-    parser.add_argument('-f',
-                        '--fuzzers',
-                        help='Fuzzers to use.',
-                        nargs='+',
-                        required=False,
-                        default=[])
-    parser.add_argument('-fc',
-                        '--fuzzer-configs',
-                        help='Fuzzer configurations to use.',
-                        nargs='+',
-                        required=False,
-                        default=[])
+    fuzzers_group = parser.add_mutually_exclusive_group()
```

Comment:
Hopefully I won't need to use this script anymore for my experiments and can just use automatic_run_experiment.py instead (I couldn't this time because there was an NFC in lafintel and there were some fuzzers afl++ asked me to run that weren't changed).